app/screens/home/RouteList.tsx

```diff
@@ -208,7 +208,11 @@ const RouteList = ({
           },
           { condition: true, label: "model.route.blockchain", value: route.blockchain },
           { condition: true, label: "model.route.asset", value: route.asset?.name },
-          { condition: true, label: "model.route.fee", value: `${route.fee}%` },
+          {
+            condition: true,
+            label: "model.route.fee",
+            value: `${route.fee}%` + (route.refBonus ? ` (${route.refBonus}% ${t("model.route.ref_bonus")})` : ""),
```

Comment:
Wait… 🤔 
You don't subtract the refBonus? By purpose?

Reply:
Why should it be subtracted?
API does any calculation needed and returns `fee` and `refBonus`